src/ds-algorithms/dictionaries.ts

```diff
@@ -0,0 +1,41 @@
+// TODO: refactor by creating a type DNode, enforcing typing across dataStore instantiation, adding, and removing
+
+class Dictionary {
+    _dataStore!: any[]
```

Comment:
Suggestion:
- use here private access modifier

Reply:
Oh, yes, yes: upon developing this class, `dataStore` I was using the private access the modifier (the _ is proof) - I will refactor immediately.